app/assets/v2/js/pages/dashboard.js

```diff
@@ -374,13 +378,15 @@ var get_search_URI = function(offset, order) {
   if (!document.hackathon) {
     if (typeof order_by !== 'undefined') {
       uri += '&order_by=' + order_by;
+      uri += '&offset=' + offset;
+      uri += '&limit=' + results_limit;
     }
   } else {
-    uri += '&event_tag=' + document.hackathon;
+    uri += `&event_tag=${document.hackathon}`;
```

Comment:
why the backticks here instead of single quotes?

Reply:
is a string literal

Reply:
ah my bad! thanks!